package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zaiusinc/node-sdk",
-  "version": "1.2.0-dev.1",
+  "version": "1.1.0-dev.1",
```

Comment:
Downgrading the version?

Reply:
I think you looked at last commit only. comparing to master the change is from 1.0.1 to 1.1.0-dev.1